common/adminBlock/EntryAddEfficiency.go

```diff
@@ -123,33 +123,41 @@ func (e *AddEfficiency) UnmarshalBinaryData(data []byte) ([]byte, error) {
 	buf := primitives.NewBuffer(data)
 	e.Init()
 
-	b, err := buf.PopByte()
+	t, err := buf.PopByte()
 	if err != nil {
 		return nil, err
 	}
 
-	if b != e.Type() {
+	if t != e.Type() {
 		return nil, fmt.Errorf("Invalid Entry type")
 	}
 
-	bl, err := buf.PopVarInt()
+	bodyLimit := uint64(buf.Len())
```

Comment:
counting the VarInt as part of the size is generally negligible, and it will only ever slightly increase the limit, which dosn't hinder the purpose of the new limits